File: Java_Basic_Jan_2023_SolutionsOfProblems/P02Ex_ConditionalStatements/P07_Shopping.java
package P02Ex_ConditionalStatements;

import java.util.Scanner;

public class P07_Shopping {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int n = Integer.parseInt(scanner.nextLine());
        int m = Integer.parseInt(scanner.nextLine());
        int p = Integer.parseInt(scanner.nextLine());

        double amount = 250 * n;
        amount = amount + m * amount * 0.35 + p * amount * 0.1;

        if (n > m) amount = amount * 0.85;

        if (budget < amount)
            System.out.printf("Not enough money! You need %.2f leva more!", amount - budget);
        else
            System.out.printf("You have %.2f leva left!", budget - amount);

    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.Ex_PB-JS-Conditional-Statements/P07_Shopping.js
function shopping(input) {
    let budget = Number(input[0]);
    let videoCards = Number(input[1]);
    let procesors = Number(input[2]);
    let rams = Number(input[3]);

    let totalCost = videoCards * 250;
    totalCost = totalCost * (1 + 0.35 * procesors + 0.1 * rams);

    if (videoCards > procesors) {
        totalCost = totalCost * 0.85;
    }
    if (budget < totalCost) {
        console.log(`Not enough money! You need ${(totalCost - budget).toFixed(2)} leva more!`);
    } else {
        console.log(`You have ${(budget - totalCost).toFixed(2)} leva left!`);
    }
}

shopping(["920.45",
    "3",
    "1",
    "1"])

